web/app.js

```diff
@@ -1330,6 +1330,78 @@ var PDFViewerApplication = {
       });
     }
   },
+
+  unbindEvents: function pdfViewBindEvents() {
```

Comment:
Since we support ES6 now, please use `unbindEvents() {` instead here (and similar in the method signature below as well).